common/templates/options.php

```diff
@@ -169,11 +169,9 @@
                         GTIN/EAN key
                         <select name="<?= $plugin->getOptionName('custom_gtin'); ?>">
```

Comment:
* Instead of the `<SELECT>`, render a loading message until it's there:
  * So that the user knows it's loading.
  * So that if the form is submitted before the load completes, the value won't be reset.
* If the loading fails, show an error message instead of the select.

Reply:
First I did that, but then I decided to disable and hide the `<select>` until load completes, instead of loading it entirely with jQuery